package.json

```diff
@@ -98,7 +98,7 @@
     "@hmcts/media-viewer": "4.0.8",
     "@hmcts/nodejs-healthcheck": "1.7.0",
     "@hmcts/properties-volume": "^0.0.13",
-    "@hmcts/rpx-xui-common-lib": "2.0.30",
+    "@hmcts/rpx-xui-common-lib": "2.0.30-COT-874-v3",
```

Comment:
No need to change, but please lower case only in lib version numbers (and branch names)